src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudServiceActor.scala

```diff
@@ -113,10 +115,8 @@ class FireCloudServiceActor extends HttpServiceActor with FireCloudDirectives
         pathPrefix("api") {
           apiRoutes
         } ~
-        pathPrefix("cookie-authed") {
-          // insecure cookie-authed routes
-          cookieAuthedRoutes
-        }
+        // insecure cookie-authed routes
+        cookieAuthedRoutes
```

Comment:
I specifically pulled this `pathPrefix` out to make the cookie-authed endpoint more consistent with our other routes. This also clarifies the test case so we don't have to guess which root path this belongs to.